ga4gh/frontend.py

```diff
@@ -172,3 +170,25 @@ def searchVariantSets():
 @app.route('/variants/search', methods=['POST', 'OPTIONS'])
 def searchVariants():
     return handleFlaskRequest(flask.request, app.backend.searchVariants)
+
+
+# The below methods ensure that JSON is returned for various errors
+# instead of the default, html
+
+
+@app.errorhandler(404)
+def pathNotFoundHandler(errorString):
+    return handleHttpError(frontendExceptions.PathNotFoundException)
```

Comment:
Probably clearer to pass in an instance of the exception here rather than the class, i.e.,
```python
return handleHttpError(frontendExceptions.PathNotFoundException())
```